OutOfSchool/OutOfSchool.WebApi/Controllers/RatingController.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Localization;
+using OutOfSchool.Services.Enums;
+using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Services;
+
+namespace OutOfSchool.WebApi.Controllers
+{
+    [ApiController]
+    [Route("[controller]/[action]")]
+    [Authorize(AuthenticationSchemes = "Bearer")]
+    public class RatingController : ControllerBase
+    {
+        private readonly IRatingService service;
+        private readonly IStringLocalizer<SharedResource> localizer;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RatingController"/> class.
+        /// </summary>
+        /// <param name="service">Service for Rating model.</param>
+        /// <param name="localizer">Localizer.</param>
+        public RatingController(IRatingService service, IStringLocalizer<SharedResource> localizer)
+        {
+            this.service = service;
+            this.localizer = localizer;
+        }
+
+        /// <summary>
+        /// Get all ratings from the database.
+        /// </summary>
+        /// <returns>List of all ratings.</returns>
+        [Authorize(Roles = "admin")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [HttpGet]
+        public async Task<IActionResult> Get()
```

Comment:
Will need to add pagination support from Yehor's PR later, as there will be a lot of records

Reply:
Close as pagination will be added to all controllers that have "Get" method once Yehor finished with PR.